src/components/Assets/UploadAssetsComponent.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Progress, Icon } from 'semantic-ui-react';
+import Dropzone from 'react-dropzone';
+import '../../_css/UploadAssets.css';
+
+const StatusMessage = props => <div className={`${props.className} `}>{props.message}</div>;
+
+const uploadStatus = (success, error) => {
+  if (success.hasOwnProperty('fail') || error) {
+    const errorIcon = (
```

Comment:
Instead of declaring a constant to be returned later. Have the following:
```jsx
return (
  <div>
    <Icon name="x icon" size="big" />
    {success.fail || error}
  </div>
);